.travis.yml

```diff
@@ -38,6 +38,9 @@ script:
   - nf-core lint ${TRAVIS_BUILD_DIR}
   # Lint the documentation
   - markdownlint ${TRAVIS_BUILD_DIR} -c ${TRAVIS_BUILD_DIR}/.github/markdownlint.yml
-  # Run the pipeline with the test profile
+  # Run the pipeline with barcoded data and mimimap2
   - nextflow run ${TRAVIS_BUILD_DIR} -profile test,docker --aligner minimap2 -ansi-log false
+  # Run the pipeline with barcoded data and graphmap
   - nextflow run ${TRAVIS_BUILD_DIR} -profile test,docker --aligner graphmap -ansi-log false
+  # Run the pipeline with non-barcoded data and mimimap2
+  - nextflow run ${TRAVIS_BUILD_DIR} -profile test_nobc,docker --aligner minimap2 -ansi-log false
```

Comment:
```suggestion
  - nextflow run ${TRAVIS_BUILD_DIR} -profile test_nonbc,docker --aligner minimap2 -ansi-log false
```

Reply:
I might just go ahead and commit this so Travis will pass and we can merge if that's okay?

Reply:
Excellent spot!